core/chains/evm/logpoller/orm.go

```diff
@@ -223,6 +223,20 @@ func (o *ORM) SelectLogsByBlockRangeFilter(start, end int64, address common.Addr
 	return logs, nil
 }
 
+// SelectLogsCreatedAfter finds logs created after some timestamp.
+func (o *ORM) SelectLogsCreatedAfter(eventSig []byte, address common.Address, after time.Time, qopts ...pg.QOpt) ([]Log, error) {
```

Comment:
Seems like there ought to be a # of confs arg here too?

In this case, there is no direct analog without a min creation time, since both `Logs()` and `LogsWithSigs()` use a block range filter.  But this and the other one being added in this PR would be the only logpoller queries with neither a block range nor a confs param.